concordium-node/Cargo.toml

```diff
@@ -67,9 +67,9 @@ url = "2.2.2"
 reqwest = { version = "0.11", features = ["stream"] }
 
 # gRPC dependencies
-tonic = "0.4.1"
-prost = "0.7.0"
-tokio = { version = "1.4.0", features = ["macros", "rt-multi-thread", "signal"] }
+tonic = "0.8"
+prost = "0.11"
+tokio = { version = "1.8.0", features = ["macros", "rt-multi-thread", "signal"] }
```

Comment:
Could we use the most recent version of Tokio instead?

Reply:
Yes, will update.

Reply:
It is now 1.20.